dissect/database/ese/tools/certlog.py

```diff
@@ -0,0 +1,154 @@
+import argparse
+import ctypes
```

Comment:
```suggestion
import datetime
```
No longer necessary, but we need datetime for the type hint.